extensions/blocks/slideshow/style.scss

```diff
@@ -236,3 +236,16 @@
 		}
 	}
 }
+
+@media only email {
+	.wp-block-jetpack-slideshow ul,
+	.wp-block-jetpack-slideshow li {
+		list-style: none;
+		margin-left: auto;
+		margin-right: auto;
+	}
+	.wp-block-jetpack-slideshow ul {
+		display: flex;
```

Comment:
Can this be done without Flexbox? I thought it wasn't well supported in email clients. 🤔

Reply:
That's what I thought, too, but I noticed there is some flex stuff happening in other converted blocks, and it appears to be working there. I haven't had a chance to test this theory yet, though, so we might end up going to tables if so.

Reply:
It depends on the style property. For instance , [`display: flex`](https://www.campaignmonitor.com/css/flexbox/display/) is better supported than [`flex-basis`](https://www.campaignmonitor.com/css/flexbox/flex-basis/).